src/include/filesystem.js

```diff
@@ -25,13 +25,13 @@ class filesystem {
     static open(file) {
         let editor = process.env.VISUAL;
 
-        if (editor || (editor = process.env.EDITOR)) {
-            return child_process.spawn(editor, [file], {
-                stdio: 'inherit'
-            });
-        } else {
-            return open(file);
+        if (!editor && !(editor = process.env.EDITOR)) {
+            editor = "vi";
         }
+
+        return child_process.spawn(editor, [file], {
+            stdio: 'inherit'
+        });
```

Comment:
Hi! Thanks for your contribution. What you suggest would not work on Windows unfortunately. What do you think of bumping the _**open**_ package from 0.0.5 to 7.0.4 and using:
```suggestion
       if (editor || (editor = process.env.EDITOR)) {
            return child_process.spawn(editor, [file], {
                stdio: 'inherit'
            });
        } else {
            return async () => await open(file);
        }
```